hw/ip_examples/im2col_spc/data/im2col_spc.hjson

```diff
@@ -0,0 +1,246 @@
+// Copyright EPFL contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "im2col_spc",
+  clock_primary: "clk_i",
+  bus_interfaces: [
+    { protocol: "reg_iface", direction: "device" }
+  ],
+  regwidth: "32",
+  registers: [
+    { name:     "SRC_PTR",
+      desc:     "Input data pointer (word aligned)",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "31:0", name: "PTR_IN", desc: "Input data pointer (word aligned)" }
+      ]
+    },
+    { name:     "DST_PTR",
+      desc:     "Output data pointer (word aligned)",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "31:0", name: "PTR_OUT", desc: "Output data pointer (word aligned)" }
+      ]
+    },
+    { name:     "IW",
+      desc:     "Image width",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "31:0", name: "SIZE", desc: "Image width" }
+      ]
+    },
+    { name:     "IH",
+      desc:     "Image heigth",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "31:0", name: "SIZE", desc: "Image heigth" }
```

Comment:
do we really need to deal with images as big as requiring 32bits? let's keep it to 16b please